include/system/termcurses.h

```diff
@@ -96,6 +96,10 @@ struct termcurses_ops_s
   CODE int (*setcolors)(FAR struct termcurses_s *dev,
                         FAR struct termcurses_colors_s *colors);
 
+  /* Set default fg/bg colors */
+
+  CODE int (*setdefaultcolors)(FAR struct termcurses_s *dev);
```

Comment:
how about we add deinit function to the restore work?

Reply:
setdefaultcolors removed.  the terminate operation has been added.